app/assets/javascripts/blueimp-gallery-video.js

```diff
@@ -56,8 +56,8 @@
         videoFactory: function (obj, callback, videoInterface) {
             var that = this,
                 options = this.options,
-                videoContainerNode = this.elementPrototype.cloneNode(false),
-                videoContainer = $(videoContainerNode),
+                videoContainerNode = document.createElement('a'),
```

Comment:
i tested this way ... didn't notice an issue whether the videoContainerNode was an `<a>` or a `<div>` ... but i think i'm going to change it back just to be safe since this isn't our code